x-pack/platform/plugins/shared/alerting/server/alerts_service/alerts_service.ts

```diff
@@ -501,4 +503,12 @@ export class AlertsService implements IAlertsService {
       ...opts,
     });
   }
+
+  public async clearAlertFlappingHistory(opts: ClearAlertFlappingHistoryParams) {
```

Comment:
Why is this here vs in the alertsClient? I guess I don't fully understand the difference between both 🤔

Reply:
I'm just following the pattern set for untracking haha